### DISCUSSION TITLE
How to visualize and save the bounding box of the trace, like the GIF of the home car of this project

### DISCUSSION THREAD

**Discussion by chris011225** [mikel-brostrom/yolo_tracking#1085] [Q&A]
Excuse me，i am green hand.How to visualize and save the bounding box of the trace, like the GIF of the  car of this project

**Comment by mikel-brostrom** [comment]
python example/track.py --show --save-mot

**Comment by chris011225** [comment]
thank you,sir




1.27
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mikel-brostrom/yolo_tracking"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月19日(星期六) 下午5:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mikel-brostrom/yolo_tracking] How to visualize and save the bounding box of the trace, like the GIF of the home car of this project (Discussion #1085)





 
python example/track.py --show --save-mot
